integration/spark2/src/test/scala/org/apache/spark/carbondata/restructure/AlterTableValidationTestCase.scala

```diff
@@ -709,6 +709,22 @@ test("test alter command for boolean data type with correct default measure valu
       Seq(Row(1))
     )
   }
+
+  test("Alter table selection in random order"){
+    sql("drop database if exists carbon cascade")
+    sql(s"create database carbon location '$dblocation'")
+    sql("use carbon")
+    sql("create table brinjal (imei string,channelsId string,gamePointId double,deviceInformationId double," +
```

Comment:
No need to create database in the test case. Use default database and also use a unique tableName...